### ISSUE TITLE
[bug] Deleting bg class from properties UI

### ISSUE THREAD

**Issue by drfarrell** [onlook-dev/onlook#920]
When a user deletes a background from the UI sidepanel, actually remove the background class. What happens is the bg class in the tailwind classes gets replaced with bg-[#00000000] which isn't the same as removing the bg entirely.

**Comment by Kitenite** [comment]
This is because we don't have a delete class option. We really should.

**Comment by alexandreantunesmnds** [comment]
Hi @Kitenite can you assign me to the issue ?

**Comment by Kitenite** [comment]
Done @alexandreantunesmnds !

**Comment by Kitenite** [comment]
@alexandreantunesmnds, how's it going? Do you need support from my end? :)

**Comment by alexandreantunesmnds** [comment]
Hi @Kitenite I am currently working on it. I think it's ok thank's for the help !

**Comment by alexandreantunesmnds** [comment]
I wanted to clarify if the issue occurs when deleting the background color on the style parameter, where the Tailwind CSS shows bg-[#00000000], but we want it to be completely removed. Is this the intended behavior we want to achieve?

Also, could you confirm if only the two files indicated need to be changed to implement this fix?

Thank you :)

**Comment by Kitenite** [comment]
@alexandreantunesmnds , yes exactly. When removing color with the UI, the CSS value for the color is being overwritten with a transparent value. It should be removed instead. 

This is only part of the problem. I think we need to also reset the CSS for the element when its tailwind class changes.

Those 2 places are the root of the implementation, yes. Usages of those functions would also need to be updated.

**Comment by sajan018** [comment]
@Kitenite  can you pls assign me this issue
